build docs test in Cirq CI failing on main

**Description of the issue**
The build docs test is failing with the following error

```
Running Sphinx v3.2.1

Sphinx version error:
The sphinxcontrib.applehelp extension used by this project needs at least Sphinx v5.0; it therefore cannot be built with this version.
FAILED
Error: Process completed with exit code 2.
```

**How to reproduce the issue**

See https://github.com/quantumlib/Cirq/actions/runs/7537252828/job/20515915318?pr=6412

--- a/rtd_docs/requirements.txt
+++ b/rtd_docs/requirements.txt
@@ -1,6 +1,14 @@
 # For generating documentation.
-# TODO(#6400,juhas) - upgrade Sphinx and remove the alabaster pin below
+
+# TODO(#6400,juhas) - upgrade Sphinx and remove the package pins below -------
 alabaster<=0.7.13
+sphinxcontrib-applehelp<=1.0.7
+sphinxcontrib-devhelp<=1.0.5
+sphinxcontrib-htmlhelp<=2.0.4
+sphinxcontrib-qthelp<=1.0.6
+sphinxcontrib-serializinghtml<=1.1.9
+# End of TODO ----------------------------------------------------------------
+
 myst-parser
 Sphinx~=3.2.0
 sphinx_rtd_theme

